app/validators/email_address_validator.rb

```diff
@@ -1,6 +1,9 @@
 class EmailAddressValidator < ActiveModel::EachValidator
+  ACCENTED = 'àèìòùÀÈÌÒÙáéíóúýÁÉÍÓÚÝâêîôûÂÊÎÔÛãñõÃÑÕäëïöüÿÄËÏÖÜŸçÇßØøÅåÆæœ'.freeze
+  EMAIL_REGEX = /\A[a-zA-Z#{ACCENTED}0-9.!\#$%&'*+\/=?^_`{|}~-]+@[a-zA-Z#{ACCENTED}0-9](?:[a-zA-Z#{ACCENTED}0-9-]{0,61}[a-zA-Z#{ACCENTED}0-9])?(?:\.[a-zA-Z0-9](?:[a-zA-Z#{ACCENTED}0-9-]{0,61}[a-zA-Z#{ACCENTED}0-9])?)*\.[a-zA-Z0-9](?:[a-zA-Z#{ACCENTED}0-9-]{0,61}[a-zA-Z#{ACCENTED}0-9])\z/i.freeze
```

Comment:
this is unreadable 😄 
please, could you make this over multiple lines with some short comments?
see this as example:
https://stackoverflow.com/questions/3762183/split-ruby-regex-over-multiple-lines

Reply:
I've added some comments and added support for foreign/accented characters 3e090a8